### PR TITLE
feat: add service.target.name

### PR DESCRIPTION
**Pull request by GraceRuan**

### CODE REVIEW

File: event-stream-processing/src/parsers/application-classification.parser.ts [hunk 1]
```diff
@@ -38,32 +40,45 @@ export class ApplicationClassificationParser implements Parser {
...
       }
     }
-    if (lodash.isNil(lodash.get(document.data, 'service.name')) && urlPath.startsWith('/clp-cgi')) {
-      lodash.set(document.data, 'service.name', 'clp-cgi');
-    }
+
     // https://www.oracle-and-apex.com/apex-url-format/
-    if (lodash.get(document.data, 'service.name') === 'apex') {
+    /* eslint-disable max-len,camelcase,@typescript-eslint/no-unsafe-call */
+    if (lodash.get(document.data, 'service.target.name') === 'apex' || lodash.get(document.data, 'labels.project') === 'apex') {
```
Comment (mbystedt) [inline event-stream-processing/src/parsers/application-classification.parser.ts:73]: I think we'll need to add in a labels.target_project and labels.source_project to the index schema. It'll need to be in a separate PR so the index can rollover before we deploy anything that sets it.

**Review by mbystedt — CHANGES_REQUESTED** [review]

File: event-stream-processing/src/parsers/application-classification.parser.ts (hunk 1 above)
Comment (GraceRuan) [reply event-stream-processing/src/parsers/application-classification.parser.ts]: parsing labels.target_project and service.target.name from url.path

**Review by mbystedt — APPROVED** [review]
